Tests/LibWeb/Text/input/script-src-set-after-insertion.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<head></head>
+<script src="include.js"></script>
+<script>
+    asyncTest((resolve, reject) => {
```

Comment:
`asyncTest()` should only have one parameter. We normally call this parameter `done`